UPGRADE-API-1.11.md

```diff
@@ -314,3 +314,11 @@
 24. `Sylius\Bundle\ApiBundle\Command\Cart\ApplyCouponToCart` and `Sylius\Bundle\ApiBundle\Command\Checkout\AddressOrder` commands have been replaced with `Sylius\Bundle\ApiBundle\Command\Checkout\UpdateCart`.
 
 25. `Sylius\Bundle\ApiBundle\CommandHandler\Cart\ApplyCouponToCartHandler` and `Sylius\Bundle\ApiBundle\CommandHandler\Checkout\AddressOrderHandler` command handlers have been replaced with `Sylius\Bundle\ApiBundle\CommandHandler\Checkout\UpdateCartHandler`.
+
+26. Following changes were made for normalization groups in `Sylius\Component\Core\Model\Order`:
+    * Removed `shop:cart:read` and `shop:cart:read` groups
+    * New groups added for item endpoints are `shop:order:read:item` and `shop:order:read:item`
+    * New groups added for collection endpoints are `shop:order:read:list` and `shop:order:read:list`
+    * Removed following fields from `Order` collection response:
+        * `channel`, `customer`, `payments`, `shipments`, `items` and  `billingAddress` fully removed
+        * from `shippingAddress` field following fields removed: `city`, `street`, `postCode`, `provinceCode`, `provinceName`
```

Comment:
billing address is the main required address, and this one should be used to provide details about person who bought items
```suggestion
        * `channel`, `customer`, `payments`, `shipments`, `items` and  `shippingAddress` fully removed
        * from `billingAddress` field following fields removed: `city`, `street`, `postCode`, `provinceCode`, `provinceName`
```